st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -474,7 +477,7 @@ def _resolve_params(action_node, original_parameters, results, chain_vars, chain
         context.update({SYSTEM_KV_PREFIX: KeyValueLookup()})
         context.update({ACTION_CONTEXT_KV_PREFIX: chain_context})
         try:
-            rendered_params = jinja_utils.render_values(mapping=action_node.params,
+            rendered_params = jinja_utils.render_values(mapping=action_node.get_parameters(),
```

Comment:
Another option would be to use `@property` / attribute getter in the `Node` class but this means we would need to change code a bit and prefix all the class variables with `_`.